pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -0,0 +1,106 @@
+
+This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.2 must be running a Kubernetes version in this range.
+
+## New features and capabilities
+
+These features and capabilities are new for Version 2.2
+
+### Upgrade catalog application via CLI and UI
+
+You can use either the CLI or the UI to [upgrade your catalog applications](../projects/applications/catalog-applications/#upgrade-catalog-applications).
+
+<p class="message--note"><strong>NOTE:</strong> Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
+
+### Better integration with VMware vSphere
+
+You can use CAPI vSphere Provider while provisioning a DKP cluster on vSphere, which allows you to manage bootstrapping of VMs on a DKP cluster. This gives you improved productivity and speed of deploying VMs on DKP in a VMWare environment, including FIPS builds and air-gapped deployments.
+
+### Zero downtime upgrades for air-gapped deployments
+
+Kubernetes Operators and other IT operations team members can use your laptop or USB drive to transfer pre-created bundles, including OS dependencies and DKP binaries into an air-gapped environment with no external connectivity. This improves the availability of the DKP air-gapped deployment and productivity of your IT operations team.
+
+### Unified user interfaces
+
+This provides a smooth experience independent of where you start your journey. A DKP Essential customer can simply update your license to access DKP Enterprise features without having to learn another interface.
+
+### Kaptain AI/ML, D2iQ’s AI/ML offering
+
+For better integrated with DKP 2.2 you can easily launch this feature as a catalog application, as well as support other platforms such as Amazon AWS, AKS, and Microsoft Azure EKS. This extends D2iQ’s openness to support Kubernetes platforms beyond DKP with Kaptain. Kaptain enables an organization to develop, deploy, and run entire ML workloads in production, at scale, with consistency and reliability.
+
+### DKP Insights
+
+This new predictive analytics tool provides greater support productivity, speed, and reduced costs. An Insights module integrated into DKP collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels. This enables you to quickly identify and resolve issues that would save a lot of time and money by not escalating to D2IQ's support. You can filter the insight summary cards by one or many insight categories for a selected cluster and project.
+
+## Component updates
+
+The following services and service components have been upgraded to the listed version:
+
```

Comment:
@gracedo - from your example, and I pulled the apps and versions down from the 2.1.1 release notes, so these do need verified/updated, please. I left your example at the top, but we can move it down to match the alphabetical order here.

Reply:
@gvasquezvargas - FYI, here's the new table.

Reply:
I will take care of updating this table in another PR that will merge into this one. Will use this jira to track https://jira.d2iq.com/browse/D2IQ-86410